### PR TITLE
Created test modal and accept cookie modal

### PR DESCRIPTION
**Pull request by lindsayessoyan**
-created a test modal 
-created the accept cookie modal

### CODE REVIEW

File: src/Components/Bake/Bake.js [hunk 1]
```diff
@@ -1,11 +1,85 @@
...
+                      </div>
+                  </div>
+              </div>
+          </div>
+      
+    )
+  }
+}
+
+class LuluModal extends Component {
```
Comment (alyssamw) [inline src/Components/Bake/Bake.js]: So excited to use these! For extensibility, could we merge AcceptModal and LuluModal into a single Modal class that can get props for what content it contains?

**Review by alyssamw — APPROVED** [review]
Hope it's okay that I waited until after finals to review this. Left a comment for code usability, but happy to merge this regardless and work it into newer tickets 👍

File: src/Components/Bake/Bake.js (hunk 1 above)
Comment (lindsayessoyan) [reply src/Components/Bake/Bake.js]: Oh definitely! I will do that asap!

**Comment by lindsayessoyan** [comment]
> Hope it's okay that I waited until after finals to review this. Left a comment for code usability, but happy to merge this regardless and work it into newer tickets 👍

No worries at all, sorry for my delayed response! That sounds great, thanks Alyssa 😀

**Pull request closed without merging**